cmd/describe/machinepool/cmd_test.go

```diff
@@ -124,11 +129,34 @@ kind: MachinePool
 `
 )
 
+type stdOutAttributes struct {
```

Comment:
@hunterkepley This looks generally useful across a lot of testing scenarios. Should we move it into the test helpers or perhaps even introduce it onto the `TestRuntime` as a method that can be easily used? WDYT?

Reply:
yeah sounds good, ill add it to the test helpers